src/app/pages/test-report/test-report.analytics.effects.ts

```diff
@@ -259,12 +293,21 @@ export class TestReportAnalyticsEffects {
       [action, tests]:
       [ReturnType<typeof drivingFaultsActions.AddDrivingFault>, TestsModel, boolean],
     ) => {
+      // TODO MES-9495 - remove old analytics
       this.analytics.logEvent(
         formatAnalyticsText(AnalyticsEventCategories.TEST_REPORT, tests),
         formatAnalyticsText(AnalyticsEvents.ADD_DRIVING_FAULT, tests),
         fullCompetencyLabels[action.faultPayload.competency],
         action.faultPayload.newFaultCount,
       );
+      // GA4 Analytics
+      this.analytics.logGAEvent(
+        formatAnalyticsText(GoogleAnalyticsEvents.ADD_FAULT, tests),
+        formatAnalyticsText(GoogleAnalyticsEventsTitles.FAULT_TYPE, tests),
+        fullCompetencyLabels[action.faultPayload.competency],
+        formatAnalyticsText(GoogleAnalyticsEventsTitles.SEVERITY, tests),
+        formatAnalyticsText(GoogleAnalyticsEventsValues.DRIVING, tests),
```

Comment:
please use the common enum for references to driving/serious/dangerous

enum ValidFaultTypes